spec/PeriodicBackgroundSync-index.bs

```diff
@@ -107,36 +107,39 @@ A <dfn>state</dfn>, which is one of "`pending`", "`firing`", "`suspended`" or "`
 The [=periodic sync scheduler=] is responsible for scheduling firing of [=periodicsync events=].
 In response to these triggers, the scheduler either schedules delayed processing to fire a [=periodicsync event=] at the appropriate time in the future, or cancels such scheduling.
 
-  <div dfn-for="periodic sync scheduler">
-    The scheduler maintains:
-    * The <dfn>time of last fire</dfn>, a [=map=], where each key is an [=origin=], and each item is a timestamp. The keys of this map are the [=origin|origins=] associated with the [=periodic sync registration/service worker registration|service worker registrations=] of [=active periodic sync registrations=]. This is initially an empty [=map=].
-  </div>
+<div dfn-for="periodic sync scheduler">
+  The [=periodic sync scheduler=] has a <dfn>time of last fire</dfn>, a [=map=], where each key is an [=origin=], and each item is a timestamp. The keys of this map are the [=origin|origins=] associated with the [=periodic sync registration/service worker registration|service worker registrations=] of [=active periodic sync registrations=]. This is initially an empty [=map=].
```

Comment:
Nit: You don't need to manually handle the plurals here

Reply:
updated everywhere.